src/modules/verifiableCredentials/verifiableCredentials.types.ts

```diff
@@ -48,3 +68,40 @@ export interface VerifyVerifiableCredentialResults {
   warnings: string[];
   errors: string[];
 }
+
+// Verifiable Presentation Types
+
+export interface SignVerifiablePresentationOptions {
+  verificationMethod?: string;
+  proofPurpose?: string;
+  domain?: Partial<TypedDataDomain>;
```

Comment:
Is `domain` needed? I think we were saying that it isn't necessary because we're not submitting to a smart contract, right?
The signature spec says that
> domain MUST have the value as defined in EIP712. If domain is not provided, a default object is applied with the only property being name with value EthereumEip712Signature2021

https://w3c-ccg.github.io/ethereum-eip712-signature-2021-spec/#suite-definition

```suggestion
```

Reply:
It's not needed, but i left an option to specify it. You think it's better to remove it?

Reply:
Yes, better to remove IMO, because we can always add later but if we want to remove it later, it is a breaking change.

Reply:
Okay, Will be removed